configure.py

```diff
@@ -470,6 +470,7 @@ def find_headers(repodir, excluded_dirs):
                 'table.cc',
```

Comment:
Why did you change this commit's subsystem tag? It definitely is not in the transport layer.

Reply:
My excessive need for unification. Now it's `system_keyspace:`

Reply:
Please also adjust the cover letter (that goes into the merge commit). I often tweet interesting commits, so it should be updated and allow readers to understand what is going on.

Reply:
To be clear, the cover letter should be stand alone and not written as a continuation from the previous attempt.

Reply:
Done!